model/model-verifier/src/test/java/io/spine/model/verify/ModelVerifierPluginTest.java

```diff
@@ -25,13 +25,13 @@
 import org.gradle.testkit.runner.BuildResult;
 import org.gradle.testkit.runner.BuildTask;
 import org.gradle.testkit.runner.TaskOutcome;
-import org.junit$pioneer.jupiter.TempDirectory;
-import org.junit$pioneer.jupiter.TempDirectory.TempDir;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.junitpioneer.jupiter.TempDirectory;
```

Comment:
@vlad-lubenskyi, why don't we use [this class](https://github.com/junit-team/junit5/blob/master/platform-tests/src/test/java/org/junit/jupiter/extensions/TempDirectory.java) natively available from JUnit guys?

Reply:
@dmitrykuzmin could you please help us on this?